elasticsearch/run.sh

```diff
@@ -133,6 +134,8 @@ verify_or_add_index_templates() {
     shopt -s failglob
     for template_file in ${ES_HOME}/index_templates/*.json
     do
+        sed -i "s,\$REPLICA_SHARDS,$REPLICA_SHARDS," $template_file
```

Comment:
@lukas-vlcek we should consider trying to make this more efficient if we only need to update 2 of the 5 templates provided. Maybe we can match on the fact that the new ones are "common."

Reply:
What about when you want to upgrade index templates or settings - that is, replace what's already there?

Reply:
@richm If we need upgrade then we need to make sure particular index templates are also pushed into ES cluster (I think they are not pushed today if they are already present). Then, as soon as new index is created in ES it will follow updated index template(s).